### PR TITLE
VBM-129: added drawer menu

### PR DESCRIPTION
**Pull request by Val-M9**
[Відео без назви - Screen Recording - 07.09.2022, 12_22_37.webm](https://user-images.githubusercontent.com/71255723/188842524-8714fae6-3845-4d2b-b56b-ef1d813b57d3.webm)

### CODE REVIEW

File: packages/mobile/src/navigation/drawer/drawer.navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { MainNavigation } from '../tabs/tabs.navigation';
+
+const HomeWithMenuNavigation: FC = () => {
+  const Drawer = createDrawerNavigator();
+
+  const options: DrawerNavigationOptions = {
+    headerShown: false,
+    swipeEdgeWidth: 200,
+    drawerStyle: {
+      width: Dimensions.get('screen').width,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/drawer.navigation.tsx:17]: Not necessary. Best practice is to leave some space

File: packages/mobile/package.json [hunk 2]
```diff
@@ -39,11 +40,13 @@
     "react-native-config": "1.4.6",
     "react-native-device-info": "10.0.2",
     "react-native-dropdown-picker": "5.4.2",
+    "react-native-gesture-handler": "^2.6.0",
```
[the remark this answers is not in the document]
Comment (Val-M9) [reply packages/mobile/package.json:44]: fixed

File: packages/mobile/src/navigation/drawer/drawer.navigation.tsx (hunk 1 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/drawer.navigation.tsx:17]: fixed

File: packages/mobile/src/navigation/tabs/tabs.navigation.tsx [hunk 3]
```diff
@@ -55,9 +55,9 @@ const MainNavigation: FC = () => {
         options={getTabOptions(MainScreenName.HOME, HomeIcon)}
       />
       <Tabs.Screen
-        name={MainScreenName.FAVORITE}
```
[the remark this answers is not in the document]
Comment (Val-M9) [reply packages/mobile/src/navigation/tabs/tabs.navigation.tsx:58]: return as it was, will rename to favourite in localization

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by dolphin323 — APPROVED** [review]

**Review by natata7 — APPROVED** [review]

**Pull request merged**